checklists/alerts.md

```diff
@@ -0,0 +1,54 @@
+# Alerts
+
+## Sentry
+
+- [ ] Exceptions that the system may raise - [How to create this alert](https://docs.sentry.io/workflow/notifications/alerts/)
+- [ ] Highly repetitive exceptions within a short time window - [How to create this alert](https://sentry.io/_/resources/customer-success/alert-rules/)
+
+## Papertrail
+
+- [ ] Error keyword - [How to create this alert](https://help.papertrailapp.com/kb/how-it-works/alerts/#create-alert)
+- [ ] Exception keyword - [How to create this alert](https://help.papertrailapp.com/kb/how-it-works/alerts/#create-alert)
+- [ ] Scheduled tasks within a time window (success/failure) - [How to create this alert](https://help.papertrailapp.com/kb/how-it-works/alerts/#inactivity-alerts)
+
+## New Relic
+
+- [ ] Response time (web) - [How to create this alert](https://docs.newrelic.com/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions)
+- [ ] Database transaction time - [How to create this alert](https://docs.newrelic.com/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions)
+- [ ] Error percentage - [How to create this alert](https://docs.newrelic.com/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions)
+- [ ] Key transactions response time - [How to create this alert](https://docs.newrelic.com/docs/apm/transactions/key-transactions/view-key-transactions-alert-information)
+- [ ] Key transactions error percentage - [How to create this alert](https://docs.newrelic.com/docs/apm/transactions/key-transactions/view-key-transactions-alert-information)
+- [ ] Third-party services (redundant with Uptime Robot)
+- [ ] Application web transaction time deviation baseline - [How to create this alert](https://docs.newrelic.com/docs/alerts/new-relic-alerts/defining-conditions/create-baseline-alert-conditions)
+- [ ] Application transaction database time deviation baseline - [How to create this alert](https://docs.newrelic.com/docs/alerts/new-relic-alerts/defining-conditions/create-baseline-alert-conditions)
```

Comment:
I see you put those here. If you think that's the only thing we need to check, then you can remove the redundant ones above.

Reply:
Don't you feel like they serve different purposes? Checking by baseline deviation and in a certain amount of time? The latter is more of a fixed threshold while the former will adapt through time.